packages/turf-line-intersect/test.ts

```diff
@@ -132,3 +132,48 @@ test("turf-line-intersect - polygon support #586", (t) => {
   t.equal(results.features.length, 1, "should return single point");
   t.end();
 });
+
+test("turf-line-intersect - self intersection behavior", (t) => {
```

Comment:
Probably no reason to have both - is there a preference for one or the other? Its not clear to me why some tests are based on fixtures - is this preferred for simple T/F style testing?

Reply:
Remembering my logic here - The reason was that the tests below specifically test the behavior of the `ignoreSelfIntersections` param. Because the in/out fixture tests all use the same setup it can't be validated in that setting.

Reply:
Fixtures were probaby chosen to avoid 1000s of lines of geojson in code. If that approach doesn't fit nicely though, by all means add normal test cases.